Knossos.NET/ViewModels/Templates/NebulaLoginViewModel.cs

```diff
@@ -55,13 +62,39 @@ public async void UpdateUI()
                     {
                         Dispatcher.UIThread.Invoke(() =>
                         {
-                            PrivateMods += mod + ", ";
+                            var item = new ListBoxItem();
+                            item.Tag = mod;
+                            item.Content = mod.ToString();
+                            PrivateMods.Add(item);
                         });
                     }
                 }
             }
         }
 
+        internal async void Install()
+        {
+            var selected = PrivateMods.FirstOrDefault(x => x.IsSelected);
+            if (selected != null)
+            {
+                if (selected.Tag is Mod mod)
+                {
+                    if (CustomLauncher.IsCustomMode && CustomLauncher.ModID != mod.id)
+                    {
+                        await MessageBox.Show(MainWindow.instance, "You can not install different mod id than the defined TC id in custom launcher mode.", "Different mod or tc", MessageBox.MessageBoxButtons.OK);
```

Comment:
Super minor, but would have a suggested change for capitalization consistency and a few specifying words. 

`You cannot install a different mod ID than the defined Total Conversion ID while in Custom Launcher mode.` 
`Different mod or Total Conversion`